lib/Backends/OpenCL/kernels.cl

```diff
@@ -328,11 +445,24 @@ size_t getNCHW(ShapeNCHW s, cl_uint32_t n, cl_uint32_t c, cl_uint32_t h,
 
 DEFINE_OPENCL_BINARY_DATA_PARALLEL_KERNEL(elementadd, float, LHS + RHS)
 DEFINE_OPENCL_BINARY_DATA_PARALLEL_KERNEL(elementsub, float, LHS - RHS)
-DEFINE_OPENCL_BINARY_DATA_PARALLEL_KERNEL(elementmul, float, LHS *RHS)
+DEFINE_OPENCL_BINARY_DATA_PARALLEL_KERNEL(elementmul, float, LHS * RHS)
```

Comment:
maybe that what clang format wanted (lhs *rhs), can you run clang-format for this PR.

Reply:
oh, this is after i run the clang-format, and it should be okay for this.